File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/response/guild/GuildListResp.java
package cn.evole.onebot.sdk.response.guild;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * @author cnlimiter
 */
@Data
public class GuildListResp {

    @SerializedName("guild_id")
    private String guildId;

    @SerializedName("guild_name")
    private String guildName;

    @SerializedName("guild_display_id")
    private String guildDisplayId;

}

File: onebot/src/main/kotlin/sdk/response/guild/GuildListResp.kt
package cn.evolvefield.onebot.sdk.response.guild

import com.google.gson.annotations.SerializedName

class GuildListResp {
    @SerializedName("guild_id")
    var guildId = ""

    @SerializedName("guild_name")
    var guildName = ""

    @SerializedName("guild_display_id")
    var guildDisplayId = ""
}
